### PR TITLE
Hotfix document corner guides

### PR DESCRIPTION
**Pull request by a-szotyori**
For portrait:
1. Default: use available height to determine the width for the guides and center horizontally.
2. If cannot fit into width: use available width to determine the height for the guides and align to preview top and don't draw button background strip.

For landscape:
1. Default: use available width to determine the height for the guides and center vertically
2. If cannot fit into height: use available height to determine the width for the guides and center horizontally.

See #116 for screenshots.

### How to test
Run on a Samsung Galaxy S6 which has a 16 to 9 ratio camera preview and on an Galaxy S5 for ex.

### Merging
Automatic

### Ticket 
#116

### CODE REVIEW

File: ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java [hunk 1]
```diff
@@ -56,13 +57,16 @@ protected void onDraw(final Canvas canvas) {
...
-        if(mPortrait) {
+        if(mShouldDrawBackgroundForButtons) {
             final Paint paintRectangle = new Paint();
             paintRectangle.setStyle(Paint.Style.FILL);
             paintRectangle.setColor(Color.BLACK);
             paintRectangle.setAlpha(75);
 
-            canvas.drawRect(mWallOffsetSide, mHeight-mWallOffsetTop -mLineLength, mWidth - mWallOffsetSide, mHeight - mWallOffsetTop, paintRectangle);
+            canvas.drawRect(mWallOffsetSide,
+                    mHeight-mWallOffsetTop -mLineLength,
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java]: Some spaces needed here between properties and `-`

File: ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java [hunk 2]
```diff
@@ -56,13 +57,16 @@ protected void onDraw(final Canvas canvas) {
...
+                    mWidth - mWallOffsetSide,
+                    mHeight - mWallOffsetBottom, paintRectangle);
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java]: `paintRectangle` should be in next line I guess.

File: ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java [hunk 3]
```diff
@@ -56,13 +57,16 @@ protected void onDraw(final Canvas canvas) {
     }
 
     private void drawBackgroundForButtons(final Canvas canvas) {
-        if(mPortrait) {
+        if(mShouldDrawBackgroundForButtons) {
```
Comment (EarlOfEgo) [inline ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java]: not sure if this should be inside the method. If I call the method with this name I expect it to draw and so it relies on something else. Better check before calling:
```Java
if(mShouldDrawBackgroundForButtons) {
    drawBackgroundForButtons(myCanvas);…
```

File: ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java [hunk 4]
```diff
@@ -71,18 +75,46 @@ protected void onMeasure(final int widthMeasureSpec, final int heightMeasureSpec
...
+            mWallOffsetBottom = mWallOffsetTop;
+            // If A4 height is greater than the view's height
+            if (mWallOffsetTop < 0) {
+                // Fit height and center horizontally
+                mWallOffsetTop = initialOffset;
+                mWallOffsetBottom = initialOffset;
+                documentHeight = mHeight - mWallOffsetTop;
+                documentWidth = (int) (documentHeight * sqrt2); //din a4
+                mWallOffsetSide = (mWidth - documentWidth) / 2;
+            }
```
Comment (EarlOfEgo) [inline ginivision/src/main/java/net/gini/android/vision/camera/FrameView.java]: could be refactored into two methods one for portrait and one for landscape.
```Java
if(portrait)
   portraitMethod… else landscapeOne
```

**Pull request merged**